gobblin-data-management/src/main/java/org/apache/gobblin/data/management/dataset/DatasetUtils.java

```diff
@@ -74,12 +74,19 @@ public Collection<CopyableFile> filter(FileSystem sourceFs, FileSystem targetFs,
    * @throws IOException
    */
   @SuppressWarnings("unchecked")
-  public static <T extends org.apache.gobblin.dataset.Dataset> DatasetsFinder<T> instantiateDatasetFinder(Properties props,
-      FileSystem fs, String default_class, Object... additionalArgs)
+  public static <T extends org.apache.gobblin.dataset.Dataset> DatasetsFinder<T> instantiateDatasetFinder(
+      Properties props, FileSystem fs, String default_class, Object... additionalArgs)
       throws IOException {
+    return instantiateDatasetFinder(DATASET_PROFILE_CLASS_KEY, props, fs, default_class, additionalArgs);
+  }
+
+  @SuppressWarnings("unchecked")
+  public static <T extends org.apache.gobblin.dataset.Dataset> DatasetsFinder<T> instantiateDatasetFinder(
```

Comment:
New method used in `DatasetFinderFilteringDecorator` to instantiate dataset finder